Formula/kafkacat.rb

```diff
@@ -29,6 +29,6 @@ def install
   end
 
   test do
-    system bin/"kafkacat", "-X", "list"
+    system bin/"kcat", "-X", "list"
```

Comment:
relates to:
- https://github.com/edenhill/kcat/commit/3fc21aa1e9facc5fa546b9c922fc26a41666a0c1